src/app/services/patient-service.ts

```diff
@@ -0,0 +1,13 @@
+import { Observable } from 'rxjs';
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+@Injectable({
+  providedIn: 'root'
+})
+export class PatientService {
+  constructor(private http: HttpClient) { }
+  PatientSearch(query: string): Observable<any> {
```

Comment:
`PatientSearch` should be `patientSearch` -> always the same, camel case for variables and methods, starts on lowercase

Reply:
done